### PR TITLE
Jupiter-30: set signup descrition for webomates test accounts

### PR DESCRIPTION
**Pull request by anuritjain**
# Description
Signup Description for Webomates Automated Tests account
Jira: https://jira.rax.io/browse/JUPITER-30

### PR Checklist before asking for +2 from peers.
- [ ] Did you include screenshots for changed UI functionality?
- [x] Did you add component tests for any new code?
- [x] Were you able to signup successfully with no console errors?
- [x] Have you deployed and validated your changes in staging?
- [x] Did you run the automated [Web-o-mates overnight tests](https://cq-portal.webomates.com/#/dashboard?product=95) on staging by clicking on the center circle in the CQ portal?
- [ ] Did you attach screenshots of Web-o-mates results in this PR?

### Before merging to master
- [x] I notified #jupiter-dev I am going to do a deployment before merging to master.

### CODE REVIEW

File: utils/signup.js [hunk 1]
```diff
@@ -50,7 +50,8 @@ export const formatRequest = (values) => {
     externalId: (values.customerInfo.productType).toUpperCase(),
     serviceLevel: 'MANAGED',
     currencyCode: values.billingInfo.currency.toUpperCase(),
-    description: `A Karate (${type.toUpperCase()}) cloud signup request from the retail site.`,
+    description: values.userInfo.accountName === 'webomates' ? 'CLOUD_TEST_SIGNUP_JUPITER'
```
Comment (abrande) [inline utils/signup.js:53]: I think it could be useful to have a couple test cases for this condition

**Review by abrande — CHANGES_REQUESTED** [review]

File: utils/signup.js (hunk 1 above)
Comment (anuritjain) [reply utils/signup.js:53]: added tests for this condition.

**Review by abrande — APPROVED** [review]

**Review by vaibhavdubey08 — APPROVED** [review]

**Review by nicko-winner — COMMENTED** [review]
Code looks looks. Please wait until we are out of moratorium to deploy it.

**Comment by nicko-winner** [comment]
Looks like smartling is not returning with a status even though they seem to have gotten the job a translators canceled it because there are no translations being added. This seems to be normal practice from what I see with other smartling jobs. It seems safe to deploy this PR anyway via Administrator privileges, but in the future it would be nice to find a way in smartling to get that status to report back correctly for jobs that don't have new translations.

**Pull request merged**